This fixes the corrupted fonts issue in dialogs.
(The preceding Q_OS_WIN conditional code does not work even in "safe mode".)

--- origsrc/stellarium-0.11.4/src/main.cpp	2012-08-26 05:32:30.000000000 -0500
+++ src/stellarium-0.11.4/src/main.cpp	2013-11-13 20:31:25.338938300 -0600
@@ -319,7 +319,7 @@ int main(int argc, char **argv)
 	// -> this seems to bring crippled arabic fonts with OpenGL2 paint engine..
 	// QFont tmpFont("Verdana");
 #else
-#ifdef Q_OS_MAC
+#if defined(Q_OS_MAC) || defined(Q_OS_CYGWIN)
 	QFont tmpFont("Verdana");
 #else
 	QFont tmpFont("DejaVu Sans");
